spec/models/broadcast_spec.rb

```diff
@@ -5,6 +5,7 @@
   it { is_expected.to validate_presence_of(:type_of) }
   it { is_expected.to validate_presence_of(:processed_html) }
   it { is_expected.to validate_inclusion_of(:type_of).in_array(%w[Announcement Welcome]) }
+  it { is_expected.to validate_uniqueness_of(:title).scoped_to(:type_of) }
```

Comment:
We might want to add a quick test or two to `spec/requests/internal/broadcasts_spec.rb` to test than an admin can't create duplicate Broadcasts, similar to how we test that you can't create [multiple active broadcasts](https://github.com/thepracticaldev/dev.to/blob/a4be185720a1e8067b54668541ca1fcb974413a4/spec/requests/internal/broadcasts_spec.rb#L91-L113).

Reply:
@vaidehijoshi: I've added a test per your request (see below file). 😄

Reply:
I LUV 2 SEE IT 👀